pdp/static/js/crmp_filters.js

```diff
@@ -27,12 +27,12 @@ function date_filter(sdate, edate) {
         filters: [
             new OpenLayers.Filter.Comparison({
                 type: ">=",
-                property: "max_obs_time",
+                property: "min_obs_time", //this was wrong in previous version, the sdate in the min obs time.
```

Comment:
Lose these changes. 

The original code only *looks* wrong. It's actually right, given that the test is whether the date ranges *overlap*. See the SO [question](http://stackoverflow.com/questions/325933/determine-whether-two-date-ranges-overlap) mentioned in the code comment just above.